pages/[profile]/[slugOrId].vue

```diff
@@ -129,16 +129,16 @@
         >
           <nuxt-link
             class="font-italic text-primary"
-            :to="`/${ artwork.creator }`"
+            :to="`/${ artwork.username }`"
```

Comment:
not everyone has an art by city username and it isn't a requirement.  also, this could be any type of username or blockchain address in the future (i.e. ANS, ARK).

still, if they have a username the url should use the username, but if not should use their address